lib/recurly/apple-pay.js

```diff
@@ -96,15 +109,33 @@ class ApplePay extends Emitter {
    */
   get lineItems () {
     // Clone configured line items
-    return [].concat(this.config.lineItems);
+    const lineItems = [].concat(this.config.lineItems);
+    return lineItems;
```

Comment:
what's the benefit of assigning the value instead of directly returning it as before?

Reply:
No benefit. This was probably an artifact of some routine I'd added and then subsequently removed here.